demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -77,6 +80,8 @@ export class DemoMeetingApp {
     this.initParameters();
     this.setMediaRegion();
     this.switchToFlow('flow-authenticate');
+    const button = document.getElementById("authenticate") as HTMLButtonElement;
```

Comment:
If we move the createMeeting logic to after clicking this button, then we dont need to disable it anymore. We should also remove the wait for button enable in the integration test.